front/src/common/BootstrapSNCF/MultiSelectSNCF.tsx

```diff
@@ -126,22 +134,22 @@ const MultiSelectSNCF = ({
 
           <div id="multiselecttoggle" className="select-menu position-relative" data-role="menu">
             {options.map((selectOption) =>
-              selectOption.label ? (
+              selectOption.group !== undefined ? (
                 <div
                   role="listitem"
                   className="select-group"
-                  key={`multiSelect-selectOption-${selectOption.label}`}
+                  key={`multiSelect-selectOption-${selectOption.group}`}
                 >
                   <div className="select-group-head">
-                    <span className="select-group-title text-uppercase">{selectOption.label}</span>
+                    <span className="select-group-title text-uppercase">{selectOption.group}</span>
                   </div>
                   <div className="select-group-content" data-role="group" data-id="0" role="list">
-                    {renderSelectToggles(selectOption.signals)}
+                    {renderSelectToggles(selectOption.options)}
                   </div>
                 </div>
               ) : (
-                <div className="select-group-content" data-role="group" data-id="0" role="list">
-                  {renderSelectToggles(selectOption.signals)}
+                <div data-role="group" data-id="0" role="list">
```

Comment:
You need to add a key to avoid a react warning.
Edit l136 like this:
```
{options.map((selectOption, index) =>
```
and then
```suggestion
                <div data-role="group" data-id="0" role="list" key={`group-${index}`}>
```